Edits to drafts/osh-1.4-rc5 to facillitate manpage generation

angelali4096 opened the pull request:
Changed main_spec.tex to maintain consistency between the names of the tex files and the names of the functions.

Got rid of extra braces in shmem_reductions.tex that were showing in the manpages.

angelali4096 committed:
Edits to drafts/osh-1.4-rc5 to facillitate manpage generation

jdinan commented on main_spec.tex:
```diff
@@ -112,7 +112,7 @@ \subsubsection{\textbf{SHPALLOC}}\label{subsec:shpalloc}
 \input{content/shpclmove.tex}
 
-\subsubsection{\textbf{SHPDEALLC}}\label{subsec:shpdealloc}
+\subsubsection{\textbf{SHPDEALLOC}}\label{subsec:shpdealloc}
```
The routine in question is actually called ```SHPDEALLC```, so the right fix here is to rename the input file rather than renaming the section.  Could you push a patch to fix this?

jdinan reviewed (changes requested):
Update SHPDEALLC bit.

angelali4096 committed:
Fixed naming of shpdeallc to be consistent

Changed filename from shpdealloc.tex to shpdeallc.tex in contents
folder. Updated main_spec.tex, table of contents, to also be con-
sistent.

jdinan commented:
Merged into #4
